plyer/platforms/linux/screenshot.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+from plyer.facades import Screenshot
+from plyer.utils import whereis_exe
+
+
+class XwdScreenshot(Screenshot):
+    def __init__(self, file_path=None):
+        default_path = 'test.jpg'

```

Comment:
Instead of `test.jpg` it could be `screenshot.jpg`. It'll be more professional and won't appear like it's still a WIP.